### DISCUSSION TITLE
FontAwesome 6 for Joomla 4.1

### DISCUSSION THREAD

**Discussion by 4digitalworld** [joomla/joomla-cms#37002] [2. Cold Features]
FontAwesome 6 is officially out. I propose to make FontAwesome 6 support for @joomla 4.1

[https://github.com/FortAwesome/Font-Awesome/releases/tag/6.0.0](https://github.com/FortAwesome/Font-Awesome/releases/tag/6.0.0)

**Comment by brianteeman** [comment]
already done https://github.com/joomla/joomla-cms/pull/36906
